docs/ko/reference/compat/array/max.md

```diff
@@ -11,14 +11,16 @@
 ## 인터페이스
 
 ```typescript
-function max<T>(items: T[]): T;
+function max<T>(items: readonly [T, ...T[]]): T;
+function max(): undefined;
+function max<T>(items?: T[]): T | undefined;
```

Comment:
```suggestion
function max<T>(items: [T, ...T[]]): T;
function max(): undefined;
function max<T>(items?: T[]): T | undefined;
```

We are not denoting `readonly` in our docs for simplicity, so let me remove this.